.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+Thanks for submitting code to Blockly!
```

Comment:
Add "Please fill out the following as part of your pull request so we can review your code more easily."

Reply:
Done.